### ISSUE TITLE
should indicate what the CBOR integers are counting since Unix epoch

### ISSUE THREAD

**Issue by TallTed** [w3c/did-core#523]
https://github.com/w3c/did-core/blob/9605a1e018dbb6b124c0b869243fe29cadd263e9/index.html#L2340-L2341

e.g. 

```
For example, CBOR-LD represents some <a>datetime</a> values
using integers to represent the number of seconds since the Unix epoch.
```

It seems likely to be appropriate to include a link to the specific section of the [CBOR RFC](https://tools.ietf.org/html/rfc8949#section-3.4.2) or the (poorly, i.e., much too subtly labeled as DRAFT, and desperately crying out for the recently implemented EDITOR'S DRAFT watermark) [CBOR-LD 1.0 spec](https://digitalbazaar.github.io/cbor-ld-spec/)

**Comment by msporny** [comment]
Applied your suggestion in 10744a0c4e098d7241cf20d26ceea15d22716ee4.

As the Editor of the CBOR-LD spec, I hesitate to link to it w/o a more permanent home for the spec. The JSON-LD CG seems to be dragging it's heels on adopting it as a work item -- there were some unresolved objections for taking on the work... perhaps it should live in the CCG (although, that would be strange). In any case, don't want to link to the CBOR-LD spec w/o a more permanent home.

Does that work for you @TallTed? Or would you rather we find a home for the spec and link to it?

**Comment by TallTed** [comment]
@msporny - Presuming that CBOR-LD simply inherits this typedef from CBOR (which seems likely as there's no mention of epoch in the draft CBOR-LD spec), I'd change the example either to --

* be about CBOR instead of CBOR-LD
* explicitly state that the CBOR-LD typedef is inherited from CBOR

-- and link to the CBOR RFC.  The first option is cleaner, but will not encourage LD, probably among other drawbacks.  The second option still _wants_ a link to the CBOR-LD spec, but could probably be left without, or with a handwave to its "anticipated future home" (which might well change between now and DID Core CR/TR).

Neither is a pure win, unless we know CBOR-LD's eventual forever home.

**Comment by jonnycrunch** [comment]
Mmm. Is CBOR-LD a thing?  And isn't this spec locked from new `features` creeping in?

**Comment by msporny** [comment]
> Mmm. Is CBOR-LD a thing? 

There is a primer:
https://docs.google.com/presentation/d/1ksh-gUdjJJwDpdleasvs9aRXEmeRvqhkVWqeitx5ZAE/edit

There is a specification:
https://digitalbazaar.github.io/cbor-ld-spec/

There is an implementation:
https://github.com/digitalbazaar/cborld

I don't know whether or not that meets your definition of "a thing".

> And isn't this spec locked from new `features` creeping in?

What new feature do you think is creeping in?

To be clear, there is no intent to add CBOR-LD to the specification in any sort of normative fashion. It's far too early in its lifetime to do that.

**Comment by TallTed** [comment]
> To be clear, there is no intent to add CBOR-LD to the specification in any sort of normative fashion. It's far too early in its lifetime to do that.

And that's our cue to change the example from CBOR-LD (which is still ephemeral, even with the draft spec and PoC implementation) to CBOR (which is solidly spec'ed in RFC and normatively referenceable).

**Comment by msporny** [comment]
> And that's our cue to change the example from CBOR-LD (which is still ephemeral, even with the draft spec and PoC implementation) to CBOR (which is solidly spec'ed in RFC and normatively referenceable).

Except that the did+cbor encoding doesn't realize the issue that the current text is attempting to point out.

While I hesitate to deal w/ the CBOR-LD reference in this issue, I've marked that this issue is ready for a PR.

**Comment by jonnycrunch** [comment]
from my understanding we are using xml string date time and not epoch time.  So not sure this is an issue for the spec.

**Comment by msporny** [comment]
The full text that we're discussing is this:

> NOTE: Representation properties lexical and value space
> It is RECOMMENDED that representations use the lexical representation of registered data types. For example, JSON and JSON-LD use the XML Schema dateTime lexical representation to represent datetimes. A representation MAY choose to represent the data types differently. For example, CBOR-LD represents some datetime values using integers to represent the number of seconds since the Unix epoch.

The text is there to clarify why the lexical representation is a SHOULD and not a MUST using a concrete example of a representation that does this today. There have been objections to reference CBOR-LD, which is fine, we can replace it with "some CBOR-based representations", but then the clarification becomes more meta.

**Comment by msporny** [comment]
Removed mention of CBOR-LD from the specification in 5b7ede42312c03d364b5b99cbf4439bfaf6216a9.

The original issue was resolved via 10744a0c4e098d7241cf20d26ceea15d22716ee4.

Closing.
